docs/integrations/enterprise-connectors/source-oracle.md

```diff
@@ -40,14 +37,42 @@ for more details.
 1. Oracle DB version 23ai, 21c or 19c.
 2. Dedicated read-only Airbyte user with access to all tables needed for replication.
 
-#### 1. Make sure your database is accessible from the machine running Airbyte
+#### 1. Make sure your database is accessible from the machine running Airbyte.
 
 This is dependent on your networking setup. The easiest way to verify if Airbyte is able to connect
 to your Oracle instance is by testing the connection in the UI.
 
-#### 2. Create a dedicated read-only user with access to the relevant tables (Recommended but optional)
+#### 2. If using CDC, make sure your database can run LogMiner and that supplemental logging is enabled for all relevant tables.
 
-This step is optional but highly recommended to allow for better permission control and auditing. Alternatively, you can use Airbyte with an existing user in your database.
+Skip this step if you don't intend to perform CDC incremental syncs.
+
+This connector relies on [Debezium](https://debezium.io/documentation/reference/stable/connectors/oracle.html) to perform CDC incremental syncs, and Debezium in turn relies on [Oracle LogMiner](https://docs.oracle.com/en/database/oracle/oracle-database/23/sutil/oracle-logminer-utility.html).
+LogMiner is typically not enabled by default and requires some extra steps to enable.
+What form these steps take depends on whether the Oracle instance is managed via Amazon RDS or not.
+
+In the case of an Amazon RDS instance, the steps are as follows:
+1. Check that `SELECT LOG_MODE FROM V$DATABASE` returns `ARCHIVELOG`. If not, ensure that backups are enabled.
+2. Execute `exec rdsadmin.rdsadmin_util.set_configuration('archivelog retention hours',24)`, possibly replacing 24 with a more suitable value depending on the intended sync frequency. For instance, if the sync is intended to be run daily, then a retention period spanning multiple days is advisable.
+3. Execute `exec rdsadmin.rdsadmin_util.alter_supplemental_logging('ADD')` to enable supplemental logging on the database.
+
+In the case of any other Oracle instance, the steps are as follows:
+1. `CONNECT ... AS SYSDBA` using [sqlplus](https://en.wikipedia.org/wiki/SQL_Plus) or equivalent.
```

Comment:
In this context I think it might be appropriate to use the trade name "SQL Plus" rather than the command itself.